README.org

```diff
@@ -453,6 +456,79 @@ For a more general index related to all-things Emacs, use [[https://github.com/e
 
     Also handles macros, functions defined through macros and let-bound variables.
 
+    If you want to jump to symbols in files which aren't loaded in your Emacs you can fallback to the more general [[https://github.com/jacktasia/dumb-jump][dumb-jump]] package.
+
+
+*** Search and Replace
+
+    In Emacs regular expressions can make use of syntax information provided by the major-mode. This allows for complex symbol searches with =isearch= and =occur=.
+    To search and jump to toplevel definitions of a buffer you can use the built-in =imenu=. The [[https://github.com/vspinu/imenu-anywhere][imenu-anywhere]] package allows to extend the scope to buffers of the same project or mode.
+    It's also worth mentioning the built-in =outline-mode= here which can be used to structure and navigate files by sections. 
+
+**** elisp-refs
+
+     [[https://github.com/Wilfred/elisp-refs][source]]
+
+     Semantic code search for Elisp which parses the code instead of doing dump text searches.
+
+
+**** el-search
+
+     [[https://elpa.gnu.org/packages/el-search.html][source]]
+
+     Lets you execute search and replace operations on symbolic expressions. For example you can search for things like defvars which don't specify an init value using the pattern `(defvar ,_)`.
+
+
+*** Refactor and Editing
+
+**** elisp-depmap
+
+     [[https://github.com/mtekman/elisp-depmap.el][source]]
+
+     Aids the refactoring process by presenting a graphical visualization of project dependencies.
+
+
+**** emacs-refactor
+
+     [[https://github.com/Wilfred/emacs-refactor][source]]
+
+     Contains various refactoring commands for Elisp.
+
+
+**** lispy
+
+     [[https://github.com/abo-abo/lispy][source]]
+
+     Contains refactoring commands and invents its own editing language for editing Lisp code in general.
+
+
+**** speed-of-thought-lisp
+
+     [[https://github.com/Malabarba/speed-of-thought-lisp][source]] provides smart way to make use of abbreviations in  Elisp.
+
+
+**** elisp-docstring
+
+     [[https://github.com/Fuco1/elisp-docstring-mode][source]]
+
+     Simiplifies editing of docstrings. Handles escaping and unexcaping of quotes automatically.
+
+
+*** Formatting
+
+**** elfmt
```

Comment:
I would put `aggressive-indent-mode` first, it's more generic and more found in the wild

Reply:
Good point, I'm biased because I don't like `aggressive-indent-mode` myself ;)